modules/ROOT/pages/about-api-catalog-cli.adoc

```diff
@@ -0,0 +1,689 @@
+
+[%header%autowidth.spread,cols="a,a"]
+|===
+| Argument | Description
+| `username` | Your Anypoint Platform username
+
+You can also pass this value using the environment variable `export ANYPOINT_USERNAME=<name>`.
+| `password` | Your Anypoint Platform password
+
+You can also pass this value using the environment variable `export ANYPOINT_PASSWORD=<pwd>`.
+|===
+
+For information about logging in using SSO, see xref:access-management::external-identity.adoc[About Identity Management].
+
+=== Client ID and Client Secret
+
+You can configure a Connected App with the `client_credentials` grant type to use with the CLI.
+
+[%header%autowidth.spread,cols="a,a"]
+|===
+| Argument | Description
+| `client_id` | The ID of the Connected App
+
+You can also pass this value using the environment variable `export ANYPOINT_CLIENT_ID=<client_id>`.
+| `client_secret` | The client secret of the Connected App
+
+You can also pass this value using the environment variable `export ANYPOINT_CLIENT_SECRET=<client_secret>`.
+|===
+
+For more information about Connected Apps, see xref:access-management::connected-apps-overview.adoc[Connected Apps].
+
+=== Environment Variable Usage
+
+Keep the following in mind while using API Catalog CLI:
+
+* Command-line options override environment variables. +
+If you don't pass a command-line Argument, the default profile properties are used.
+* If not specified, the default environment is production.
+
+=== Timeout
+
+Your Anypoint session expires based on the *Default session timeout* configured in your Organization settings.
+
+// Authentication end 
+
+== Create a Descriptor File Manually
+
+You can optionally create a descriptor file manually rather than using the `api-catalog create-descriptor` command to create it. 
+
+After you create the descriptor file, copy it to the file path or repository where your API projects are stored. 
+
+The descriptor file must start with the following:
+
+---
+
+ #%Catalog Descriptor 0.6
+
+---
+
+TIP: Run the `api-catalog publish-asset` command using the `--dry-run` option to ensure your manually-created descriptor file is valid. 
+
+=== Descriptor YAML Schema
+
+Following are the YAML tags used to describe the APIs you want to catalog. 
+
+[%header,cols="20a,80a"]
+|===
+| Tag | Description 
+
+| triggerConditions | This section enables you to set conditions to restrict the APIs that are included when the `api-catalog publish-asset` command is run.   
+
+This example sets `triggerConditions` to enable the `api-catalog publish-asset` command to set conditions to publish APIs for just a designated branch or reference, tagged in a certain way, and authored by a specific user. 
+
+---
+ branches:
+   - main
+   - release/(.*)
+ tags:
+   - support
+   - release/(.*)
+ user:
+   - admin
+
+---
+
+| contact | The contact name and email ID.   
+
+| apis | A list of the APIs to be cataloged. 
```

Comment:
When I look at the table, I have to scroll to get to this column entry, because there are so many entries in the column to its right. This is way too much info for that column. Let's think about ways to remove the list of names and descriptions from the larger table.